tests/src/test/java/com/example/CharFuzzer.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueLow;
+import com.code_intelligence.jazzer.mutation.annotation.InRange;
+
+public class CharFuzzer {
+  public static void fuzzerTestOneInput(@InRange(min = '\u0000', max = '\uFFFF') char data) {
```

Comment:
I'd limit the range and add an assert that `data` is inside the specified range.